design/root.go

```diff
@@ -0,0 +1,35 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var root = a.Type("Root", func() {
+	a.Description("JSONAPI store for the data of a Root.")
+	a.Attribute("relationships", a.HashOf(d.String, d.Any), "Describes relationship between names and links")
+	a.Attribute("type", d.String, func() {
+		a.Enum("endpoints")
+	})
+	a.Attribute("id", d.UUID, "ID of root (this is a newly generated UUID upon every call)")
```

Comment:
This UUID would change on every call to `/root` endpoint. Why do we even need it?

Reply:
To comply to JSONAPI: http://jsonapi.org/format/#document-resource-objects

Reply:
Okay.